src/app/lib/config/services/arabic.ts

```diff
@@ -276,6 +276,68 @@ export const service: DefaultServiceConfig = {
       },
       topStoriesTitle: 'الأخبار الرئيسية',
       featuresAnalysisTitle: 'اخترنا لكم',
+      ugc: {
+        // No JavaScript
+        noJsHeading: undefined,
+        noJsDescription: undefined,
+
+        // Optional
+        optional: 'اختياري',
+
+        // File upload
+        fileUploadLiveRegionText: undefined,
+        fileUploadLiveRegionUpdateText: undefined,
+        fileUploadListHeading: 'هذا ما تحمّله:',
+        fileUploadButton: 'اختر ملفا',
+        fileUploadRemoveButton: undefined,
+
+        // Submit button
+        submitButton: 'إرسال',
+
+        // Validation
+        validationRequired: 'رجاء املأ هذه الخانة الإلزامية.',
+        validationInvalidEmail:
+          'هذا لا يبدو صحيحا. يرجى إدخال بريد إلكتروني صحيح.',
+        validationInvalidTelephone: undefined,
+        validationFilesTooMany:
+          'لقد أضفت ملفات أكثر من المسموح. الحد الأقصى لعدد الملفات {{maxFiles}}.',
+        validationFilesNotEnough:
+          'لا توجد ملفات كافية. يرجى إضافة {{minFiles}} على الأقل.',
+        validationFilesSizeExceeded:
+          'عذرا، حجم هذه الملفات أكبر من المسموح. يمكنك تحميل ملفات يصل حجمها إلى 1.2 غيغابايت في المرة الواحدة.',
+        validationFilesTooSmall: 'هذا الملف لا يصلح. جرب ملفا آخر.',
+        validationFilesInvalidType:
+          'عذرا، لا يمكننا استقبال هذا النوع من الملفات. يرجى إضافة ملفات من الأنواع التالية {{fileTypes}}.',
```

Comment:
Just wondering whether we need some extra logic for this {{xxx}} templating script to work in Simorgh? 

E.g. if I go to the [Storybook permalink in arabic](https://wsteam1-971-add-uploader-translations--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/pages-ugc-page--form-with-file-upload&globals=service.service:arabic)and upload a PDF then I can see it says {{fileTypes}}:

![Screenshot 2024-07-02 at 14 54 42](https://github.com/bbc/simorgh/assets/98817636/6f3a9437-1450-4c01-b816-b68fb46672fd)

Reply:
Edit: Realised we are handling quite a few of these in the code below, perhaps we missed the File Upload Error ones within the scope of that work

Reply:
Taking a closer look it seems like the only ones we're missing logic for are:
```
maxFiles
minFiles
fileTypes
```

Reply:
I've added logic for this now in https://github.com/bbc/simorgh/pull/11742/commits/5339486cae3063bd92ed5a8d88015e4e3db43d5b. Can't think of another way to it off the top of my head, as we need the translation and validation values, so it can't really be done in those `validateFunctions.ts` functions.